pkg/solana/logpoller/loader.go

```diff
@@ -59,6 +58,7 @@ func NewEncodedLogCollector(
 			return []services.Service{c.workers}
 		},
 	}.NewServiceEngine(lggr)
+	c.lggr = c.engine
```

Comment:
Engine embeds a SugaredLogger, and includes the service name `EncodedLogCollector`